app/src/main/java/com/vultisig/wallet/ui/models/deposit/DepositFormViewModel.kt

```
@@ -2806,7 +2818,7 @@ constructor(
             transform(decimalValue, price).toPlainString()
         } catch (e: CancellationException) {
             throw e
-        } catch (_: Exception) {
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)